scratchpad/cctx_encode.py

```diff
@@ -35,7 +34,7 @@
     packet = cc.encode(frame)
     print('   ', packet)
 
-    fh.write(str(buffer(packet)))
+    fh.write(str(memoryview(packet)))
```

Comment:
This actually looks wrong, the file handle should have been opened with `open('test.flv', 'wb')` and then writing is just `fh.write(bytes(packet))`.